luigi/server.py

```diff
@@ -273,6 +273,10 @@ class RootPathHandler(BaseTaskHistoryHandler):
     def get(self):
         self.redirect("/static/visualiser/index.html")
 
+    def head(self):
```

Comment:
Is the name "head" a convention for health checking? 

Regardless it caught me by surprise, maybe add some docs to this function on why it exist? It should also say what 204.

Reply:
"head" is for the HEAD http request, and it's not a convention i would say, but it's what fits the purpose in the protocol. 
I can add some docs if you want, but i don't know where to put it, there's only autogenerated docs for the web API.

Reply:
What about `""" function docs """`?